src/stream.js

```diff
@@ -1,28 +1,94 @@
 'use strict';
 
 const logger = require('./logger');
-const groupBy = require('lodash.groupby');
+const _ = require('lodash');
 const AWS = require('aws-sdk');
+const blockKit = require('./blockKit');
+const slack = require('./slackClient');
+const dynamo = require('./dynamo');
+
 const allowedEvents = ['MODIFY', 'INSERT'];
 
 module.exports.handler = async (event) => {
+	logger.debug({event}, 'event');
 	const unmarshalledRecords = event.Records
 		.filter(record => allowedEvents.includes(record.eventName))
-		.map(record => AWS.DynamoDB.Converter.unmarshall(record.dynamodb.NewImage));
+		.map(record => AWS.DynamoDB.Converter.unmarshall(record.dynamodb.NewImage))
+		.filter(record => {
+			if (!record.channel || !record.date) {
+				logger.warn({record}, 'Bad Record');
+				return false;
+			}
+
+			// Tag for a transient record that should be ignored.
+			if (record.date === '_current') {
+				return false;
+			}
 
-	const groupedRecords = groupBy(unmarshalledRecords, record => {
-		if (!record.channel || !record.date) {
-			logger.warn({record}, 'Bad Record');
-			return 'deleteMe';
-		}
+			// ignore cases where there is a userlist or an answer without a slackId
```

Comment:
This comment just explains what the code does. How about a comment that explains why we have this code (i.e. to handle the race condition between setting the slack ID and voting/answering).